src/JsonRpcEngine.ts

```diff
@@ -259,10 +270,14 @@ export class JsonRpcEngine extends SafeEventEmitter {
    * error and the response object to the given callback.
    *
    * Does not reject.
+   *
+   * @param callerReq - The request object from the caller.
+   * @param callback - The callback function.
+   * @returns Nothing.
```

Comment:
Hmm. Usually this gets omitted when we return nothing. Did the linter request this?

Reply:
Yep. I was surprised by this too. Probably a bug in the JSDoc plugin.

Reply:
I guess even for a `Promise<void>` return, it might be useful saying _when_ the function returns. In case it does some work after the return. That's a thing we do sometime, rarely.